### ISSUE TITLE
Add MacPorts as an option to install, do not require Homebrew

### ISSUE THREAD

**Issue by barracuda156** [vmagnin/gtk-fortran#287]
@vmagnin I have added the port for this: https://ports.macports.org/port/gtk-fortran

And independently, perhaps it is better not to require Homebrew (or any package manager), since everything can be built from source without package managers, if desired.
I refer to "You need the Homebrew package manager" here: https://github.com/vmagnin/gtk-fortran/wiki/Installation

**Comment by vmagnin** [comment]
Thanks a lot @barracuda156 for that contribution!
Do you envision also a gtk4 branch version?

Concerning the installation instructions, I am not familiar with macOS. If you can help reformulating that whole section for the benefit of mac users, your help would be welcome.

**Comment by barracuda156** [comment]
@vmagnin I apologize, I did not address installation instructions part. Is it still relevant?

**Comment by vmagnin** [comment]
@barracuda156 
yes, if you have time to review and update the macOS installation section, it would be helpful:
https://github.com/vmagnin/gtk-fortran/wiki/Installation#macos

You can just post it here and I will update the Wiki page. I don't know if you can have access to the markdown code, but it is short and it is not important if you can't.

Thanks!
